Application/EdFi.Ods.Common/Composites/CompositeDefinitionProcessorContext.cs

```diff
@@ -66,10 +66,6 @@ public class CompositeDefinitionProcessorContext
 
         public bool ShouldIncludeResourceSubtype() => AttributeValueAsBool(CompositeDefinitionHelper.IncludeResourceSubtype);
 
-        public bool ShouldUseHierarchy() => AttributeValueAsBool(CompositeDefinitionHelper.UseHierarchy);
```

Comment:
Is there a custom attribute that needs to be removed?

Reply:
As in a .NET custom attribute? No.

For reference:
```private bool AttributeValueAsBool(string attributeName)
{
    bool.TryParse(AttributeValue(attributeName), out bool result);
    return result;
}
```
... and ...

```
public string AttributeValue(string attributeName) => CurrentElement.AttributeValue(attributeName);
```

This code is extracting the attribute that was removed from the composites definition XML schema.